packages/@n8n/config/src/configs/database.config.ts

```diff
@@ -183,4 +183,10 @@ export class DatabaseConfig {
 
 	@Nested
 	sqlite: SqliteConfig;
+
+	sanitize() {
+		if (this.type === 'mariadb' || this.type === 'mysqldb') {
+			throw new Error('MySQL and MariaDB have been removed. Please migrate to PostgreSQL.');
```

Comment:
We don't want this in Sentry

```suggestion
			throw new UserError('MySQL and MariaDB have been removed. Please migrate to PostgreSQL.');
```

Reply:
Importing UserError would create a circular dependency. Added a custom error as a workaround